Dockerfile228

```diff
@@ -1,4 +1,5 @@
-FROM quay.io/pypa/manylinux_2_28_x86_64
+ARG ARCH=x86_64
```

Comment:
@ccrutchf I am a bit confused by this, can this line just be removed since the ``ARCH`` variable  is being defined from the outside environment (as far as I have understood)?

Reply:
I just did some reading, the line is necessary, but the value is not.  We can leave the default as null.